docs/addons/calico.md

```diff
@@ -0,0 +1,142 @@
+# Calico Add-on
+
+The `Calico` addon adds support for Kubernetes Network Policies to an EKS cluster.
+
+[Project Calico](https://www.projectcalico.org/) is an open source networking and network security solution for containers, virtual machines, and native host-based workloads. To secure workloads in Kubernetes, Calico utilizes Network Policies as you will see below.
+
+## Usage
+
+```typescript
+import { addons }  from '@shapirov/cdk-eks-blueprint';
+
+const myCalicoCNI = new addon.CalicoAddon();
+const addOns: Array<ClusterAddOn> = [ myCalicoCNI ];
+
+const app = new cdk.App();
+new EksBlueprint(app, 'my-stack-name', addOns, [], {
+  env: {
+      account: <AWS_ACCOUNT_ID>,
+      region: <AWS_REGION>,
+  },
+});
+```
+## Securing your environment with Kubernetes Network Policies
+
+By default, native VPC-CNI plugin for Kubernetes on EKS does not support Kubernetes Network Policies. Installing Calico addon (or alternate CNI provider) for network policy support will enable customers to define and apply standard [Kubernetes Network Policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/). 
+
+However, Calico also allows Custom Resource Definitions (CRD) which gives you the ability to add features not in the standard Kubernetes policies, such as:
+- Explicit Deny rules
+- Layer 7 rule support (i.e. Http Request types)
+- Endpoint support other than standard pods: OpenShift, VMs, interfaces, etc. 
+
+In order to use CRDs (in particular defined within the *projectcalico.org/v3* Calico API), you need to install the Calico CLI ([calicoctl](https://docs.projectcalico.org/getting-started/clis/calicoctl/install)). You can find more information about Calico Network Policy and using `calicoctl` [here](https://docs.projectcalico.org/security/calico-network-policy). 
+
+In this section, you will look at how the standard Kubernetes Network Policies are applied. This will be done using `kubectl`.
+
+### Pod to Pod communications with no policies
+
+If you deploy [App of Apps installed using ArgoCD](https://github.com/aws-quickstart/quickstart-ssp-amazon-eks/blob/feature/calico/docs/getting-started.md#deploy-workloads-with-argocd), you can verify that there are no network policies in place:
+
+```bash
+kubectl get networkpolicy -A
+```
+
+This means that any resources within the cluster should be able to make ingress and egress connections with other resources within and outside the cluster. You can verify, for example, that you are able to ping from `team-riker` pod to `team-burnham` pod.
```

Comment:
Based on the security groups defined: 1/ node level 2/ explicit security groups that customers can associate with the workloads.

Reply:
Overall, looks very good, approving.